app.R

```diff
@@ -1830,6 +1830,19 @@ select count(nct_id) as number_sites, nct_id from trial_sites where org_status =
       df_crit$biomarker_api_inc_matches <- NA 
     }
     #browser()
+
+    # Load prior therapy previously saved by ETL job.
+    trials_with_prior_therapy_inc_df <- safe_query(dbGetQuery,
+      "select nct_id, nci_thesaurus_concept_id from trial_prior_therapies where eligibility_criterion='inclusion'")
+    trials_with_prior_therapy_exc_df <- safe_query(dbGetQuery,
+      "select nct_id, nci_thesaurus_concept_id from trial_prior_therapies where eligibility_criterion='exclusion'")
+
```

Comment:
This two SQL statements should also add 

`inclusion_indicator = 'TRIAL' `  to their WHERE clauses.  The 'TREE" ones are just rollups for that trial (i.e. ancestors going up the DAG)